### DISCUSSION TITLE
Alist域名在微信中被封禁

### DISCUSSION THREAD

**Discussion by DKA001** [AlistGo/alist#3972] [Q&A]
封禁提示：该网页可能存在被他人恶意利用生成违规内容的情况（如xss注入、文件上传漏洞等）。请点击“申请恢复访问”按钮，根据页面提供的链接进行排查，修复漏洞后可申请恢复访问。

**Comment by Rektyfikowany** [comment]
官方网站`https://alist.nn.ci`测试微信访问正常，可以写出遇到问题的具体域名吗

**Reply by DKA001** [reply]
不好意思，我没有表达清楚。是我们自己搭建好的alist域名，昨天开始测试，然后今天就提示域名封禁。因为我们是内部使用的，就不方便放域名了。看了网上其他alist站点，在微信里打开也有这个封禁提示。

**Reply by Rektyfikowany** [reply]
随机测试了几个公开站，部分正常部分被封，域名问题还是建议联系腾讯官方，微信是会有些额外限制

**Reply by Gokou-Ruri** [reply]
你自己的域名被微信封了关alist啥事？
